app/poros/following.rb

```diff
@@ -1,8 +1,20 @@
 class Following
-  attr_reader :login, :html_url
+  attr_reader :login,
+              :html_url
 
   def initialize(attributes)
     @login = attributes[:login]
     @html_url = attributes[:html_url]
   end
+
+  #should probably be refactored into UserInfo
```

Comment:
Agreed. There is some repeated code here between Follower and Following that could be pulled out.